### PR TITLE
Adding ability to use PTY when executing SSH commands

### PR DESCRIPTION
**Pull request by njenan**
Adding the ability to execute ssh commands as PTY.  This is useful when trying to run sudo commands on linux distributions that do not allow non-TTY sudo.

I saw there was French localization files in the repo, but unfortunately I do not know the correct translations.  I've left them as `???` and I would be very grateful for some help here!

### CODE REVIEW

File: src/main/resources/org/apache/jmeter/protocol/ssh/sampler/SSHCommandSamplerResources_fr_FR.properties [hunk 1]
```diff
@@ -31,6 +31,8 @@ connectionTimeout.displayName=Expiration de la connexion
 connectionTimeout.shortDescription=Temps a attendre avent de couper la connexion (ms)
 useReturnCode.displayName=Utiliser le code retour
 useReturnCode.shortDescription=Un code retour non nul fait \u00e9chouer le test
+useTty.displayName=???
```
Comment (yciabaud) [inline src/main/resources/org/apache/jmeter/protocol/ssh/sampler/SSHCommandSamplerResources_fr_FR.properties]: French: Utiliser TTY

File: src/main/resources/org/apache/jmeter/protocol/ssh/sampler/SSHCommandSamplerResources_fr_FR.properties [hunk 2]
```diff
@@ -31,6 +31,8 @@ connectionTimeout.displayName=Expiration de la connexion
...
+useTty.shortDescription=???
```
Comment (yciabaud) [inline src/main/resources/org/apache/jmeter/protocol/ssh/sampler/SSHCommandSamplerResources_fr_FR.properties]: French: Utiliser TTY pour établir la connexion SSH

**Comment by njenan** [comment]
All right, I added the localization, thanks for your help with the translations!

**Comment by yciabaud** [comment]
Just merged it, thank you again for sharing!

**Comment by yciabaud** [comment]
Released in [v1.1.1](https://github.com/yciabaud/jmeter-ssh-sampler/releases/tag/jmeter-ssh-sampler-1.1.1)

**Pull request merged**